blueflood-http/src/main/java/com/rackspacecloud/blueflood/tracker/Tracker.java

```diff
@@ -123,9 +123,14 @@ public static void track(HttpRequest request) {
         }
     }
 
-    public static void trackDelayedMetricsTenant(String tenantid) {
+    public static void trackDelayedMetricsTenant(String tenantid, List<String> metricNames) {
         if (isTrackingDelayedMetrics) {
-            String logMessage = String.format("[TRACKER][DELAYED METRIC] Tenant sending delayed metric is %s",tenantid);
+            StringBuilder sb = new StringBuilder();
+            for(String name : metricNames) {
+                sb.append(name);
+                sb.append(";");
+            }
+            String logMessage = String.format("[TRACKER][DELAYED METRIC] Tenant sending delayed metric is %s with the delayed metrics -- %s",tenantid,sb.toString());
```

Comment:
@tilogaat  let's make this a new story, I am going to +1 this PR once you answered the previous questions